src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -258,10 +265,12 @@
 short-summary: Copies a blob asynchronously. Use `az storage blob show` to check the status of the blobs.
 examples:
   - name: Copies a blob asynchronously. Use `az storage blob show` to check the status of the blobs. (autogenerated)
-    text: az storage blob copy start --account-key 00000000 --account-name MyAccount --destination-blob MyDestinationBlob --destination-container MyDestinationContainer --source-uri https://storage.blob.core.windows.net/photos
+    text: |
```

Comment:
duplicate examples

Reply:
I guess not, the below example has an additional parameter `--sas-token $sas`